### PR TITLE
fix(validation-messages): fix overlap with long message

### PR DESCRIPTION
**Pull request by braddialpad**
## Description

Had a team asking about this so decided to prioritize it since it's a pretty small fix
https://dialpad.atlassian.net/browse/DT-1024

Set min-width in addition to width on validation messages, otherwise the content expands to overlap the icon when there is a multiline message. This is because `background-image` is being used for the icon so it is not recognized as actual content within the element.

before:
Screenshot 2023-03-24 at 12 15 32 PM

after:
Screenshot 2023-03-24 at 12 15 38 PM

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: lib/build/less/components/forms.less [hunk 1]
```diff
@@ -114,7 +114,8 @@ fieldset {
   // Icon Slot
   &::before {
     display: block;
-    width: calc(var(--size-500) + var(--size-200)); // 16
```
Comment (braddialpad) [inline lib/build/less/components/forms.less:117]: @francisrupert this is 18 not 16 like the comment says. Is this intentional? the icon is 16x16.

Comment (francisrupert) [reply lib/build/less/components/forms.less:117]: Yeah, the `::before` is meant to be 16x16.

**Comment by francisrupert** [comment]
> This is because background-image is being used for the icon...

And that's an outdated method of building this kind of thing. But swallowing that in the meantime since we're hoping all of these form-like components would be rebuilt (even though that also might be a while).

**Review by francisrupert — APPROVED** [review]

**Comment by braddialpad** [comment]
> And that's an outdated method of building this kind of thing. But swallowing that in the meantime since we're hoping all of these form-like components would be rebuilt (even though that also might be a while).

Yep I agree. I was tempted to restructure this whole thing but held myself back for a quick fix for now.

**Pull request merged**